docs/contributing/testing.rst

```diff
@@ -135,6 +138,34 @@ commands to run tests and obtain (html) coverage report:
 
 The output is available in ``$(pwd)/pythoncov``.
 
+C++
+-----
+
+Include the following options as part of your build command:
+
+.. code-block:: bash
+
+   -DKATANA_ENABLE_COVERAGE=ON -DCMAKE_BUILD_TYPE=Debug
+
+After the build is done, binaries will be instrumented to collect
+profiling data.
+
+Now you can run any test that you wish.  For example:
+
+.. code-block:: bash
+
+   ctest -L quick
+
+There are several tools you can use to process obtained profiling data
+after test run.  One option is gcovr.  You can obtain report by
```

Comment:
It would be nice to specify what is/will be used to generate the automatic reports. That way people can get output that is familiar.

Reply:
Note that I do mention gcovr and I give an example command.  Did I misunderstand your comment?

Reply:
You mention it, but you just say "one option" you don't say that it's the one used to generate the nightly reports.

Reply:
That is correct; I was trying to keep open and closed separate here.  Internal script contains the entire command with adjustments.

Reply:
More information is good in my book. And the script will be in this repo as well eventually I trust.